### PR TITLE
CTNXT-646: Refactor create react component command

### PR DESCRIPTION
**Pull request by NataliaMarkaki**

### CODE REVIEW

File: cli/templates/component-react/_test-ut/react-component-test.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import chai from 'chai';
-import { mount, shallow } from 'enzyme';
-import chaiEnzyme from 'chai-enzyme'
 import React from 'react';
+import {
+			renderIntoDocument,
```
Comment (briandipalma) [inline cli/templates/component-react/_test-ut/react-component-test.js]: Is this formatted with prettier? If you run `npm i` at the root of `caplin-dev-tools` it should install ESLint and if your editor has ESLint/prettier integration it should all be handled for you.

File: cli/templates/component-react/react-component.js [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
       count: 0
-    };
-  },
+    }
+  }
+
   onClick() {
-    this.setState({count: this.state.count + 1});
-  },
+    this.setState({ count: this.state.count + 1 });
```
Comment (briandipalma) [inline cli/templates/component-react/react-component.js]: Should probably use the function argument setState approach. https://facebook.github.io/react/docs/react-component.html#setstate it's the recommended/preferred/safer way when the state is based on the previous state.

File: cli/templates/component-react/_test-ut/react-component-test.js (hunk 1 above)
Comment (NataliaMarkaki) [inline cli/templates/component-react/_test-ut/react-component-test.js]: I have prettier installed but when I copied this part from a different file it added the spaces without me noticing. For some reason prettier didn't automatically correct it.

Comment (briandipalma) [inline cli/templates/component-react/_test-ut/react-component-test.js]: Do you have an editor plugin for prettier? Is your editor set up for format on save?

Comment (NataliaMarkaki) [inline cli/templates/component-react/_test-ut/react-component-test.js]: I use sublime and it does have a plugin. It does reformat the code on save but this time somehow it didn't. I tried manually (right-click and choose JsPrettier) and it still didn't change it.

**Comment by briandipalma** [comment]
OK, there might be a bug in the sublime addon. No worries, it looks all good to me.

**Pull request merged**